code/game/objects/structures/dresser.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "dresser"
 	density = 1
 	anchored = 1
+	burn_state = FLAMMABLE
+	burntime = 25
```

Comment:
Losing the ability to completely change your underwear in what, 2% of rounds is hardly a showstopper, imo @Fox-McCloud